src/testing/test-helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { AxePuppeteer } from '@axe-core/puppeteer';
+
+const ignoredDefaults = ['document-title', 'html-has-lang', 'page-has-heading-one', 'landmark-one-main', 'region'];
+
+const buildAxeErrorOutput = violations =>
+  violations
+    .map(
+      v =>
+        `[${v.impact}] ${v.help}
+Id: ${v.id}
+See ${v.helpUrl}
+${v.nodes.reduce((str, { html, target }) => [str, html, ...target].join('\n\n'), '')}`,
+    )
+    .join('\n') || null;
+
+export async function axeCheck(page: any, additionalIgnored: string[] = []) {
+  const ignoredRules = [...ignoredDefaults, ...additionalIgnored];
+  const results = await new AxePuppeteer(page).disableRules(ignoredRules).analyze();
+
+  const axeError = buildAxeErrorOutput(results.violations);
+  expect(axeError).toBe(null);
```

Comment:
I went with allowing `moderate` and `minor` impact violations to cause errors in addition to `serious` and `critical`.

Reply:
This seems fine to me for now. If it becomes a problem later, we can reassess.

Reply:
Good call! Even `moderate` and `minor` can cause CI breaks upstream, so better to catch them in components.